tests/serverpod_test_server/lib/src/generated/object_with_uri.dart

```diff
@@ -0,0 +1,201 @@
+    _i1.Session session, {
+    ObjectWithUriExpressionBuilder? where,
+    int? limit,
+    int? offset,
+    _i1.Column? orderBy,
+    List<_i1.Order>? orderByList,
+    bool orderDescending = false,
+    bool useCache = true,
+    _i1.Transaction? transaction,
+  }) async {
+    return session.db.find<ObjectWithUri>(
+      where: where != null ? where(ObjectWithUri.t) : null,
+      limit: limit,
+      offset: offset,
+      orderBy: orderBy,
+      orderByList: orderByList,
+      orderDescending: orderDescending,
+      useCache: useCache,
+      transaction: transaction,
+    );
+  }
+
+  static Future<ObjectWithUri?> findSingleRow(
+    _i1.Session session, {
+    ObjectWithUriExpressionBuilder? where,
+    int? offset,
+    _i1.Column? orderBy,
+    bool orderDescending = false,
+    bool useCache = true,
+    _i1.Transaction? transaction,
+  }) async {
+    return session.db.findSingleRow<ObjectWithUri>(
+      where: where != null ? where(ObjectWithUri.t) : null,
+      offset: offset,
+      orderBy: orderBy,
+      orderDescending: orderDescending,
+      useCache: useCache,
+      transaction: transaction,
+    );
+  }
+
+  static Future<ObjectWithUri?> findById(
+    _i1.Session session,
+    int id,
+  ) async {
+    return session.db.findById<ObjectWithUri>(id);
+  }
+
+  static Future<int> delete(
+    _i1.Session session, {
+    required ObjectWithUriExpressionBuilder where,
+    _i1.Transaction? transaction,
+  }) async {
+    return session.db.delete<ObjectWithUri>(
+      where: where(ObjectWithUri.t),
+      transaction: transaction,
+    );
+  }
+
+  static Future<bool> deleteRow(
+    _i1.Session session,
+    ObjectWithUri row, {
+    _i1.Transaction? transaction,
+  }) async {
+    return session.db.deleteRow(
+      row,
+      transaction: transaction,
+    );
+  }
+
+  static Future<bool> update(
+    _i1.Session session,
+    ObjectWithUri row, {
+    _i1.Transaction? transaction,
+  }) async {
+    return session.db.update(
+      row,
+      transaction: transaction,
+    );
+  }
+
+  static Future<void> insert(
+    _i1.Session session,
+    ObjectWithUri row, {
+    _i1.Transaction? transaction,
+  }) async {
+    return session.db.insert(
+      row,
+      transaction: transaction,
+    );
+  }
+
+  static Future<int> count(
+    _i1.Session session, {
+    ObjectWithUriExpressionBuilder? where,
+    int? limit,
+    bool useCache = true,
+    _i1.Transaction? transaction,
+  }) async {
+    return session.db.count<ObjectWithUri>(
+      where: where != null ? where(ObjectWithUri.t) : null,
+      limit: limit,
+      useCache: useCache,
+      transaction: transaction,
+    );
+  }
+}
+
+typedef ObjectWithUriExpressionBuilder = _i1.Expression Function(
+    ObjectWithUriTable);
+
+class ObjectWithUriTable extends _i1.Table {
+  ObjectWithUriTable() : super(tableName: 'object_with_uri');
+
+  final id = _i1.ColumnInt('id');
+
+  final url = _i1.ColumnSerializable('url');
```

Comment:
I think a `ColumnString` would be better.

Change would go here.
https://github.com/serverpod/serverpod/blob/046b95537404f340784fbef9a15d1ca62b6e332f/tools/serverpod_cli/bin/generator/types.dart#L173-L186